2025.04.04.html

```diff
@@ -19,11 +19,11 @@
 <p>The first quarter of 2025 is over, and despite the fact that we are a little bit behind the schedule, we are still on track to release the new version of BOINC soon.
 We took some additional time to finalize the support of the new <a href="2024.12.01.html">BUDA  functionality</a>, which is a new way of distributing science applications to our users.
 With the huge help of the <a href="https://lhcathomedev.cern.ch/lhcathome-dev/">LHC@Home Development Project</a> we were able to test the new functionality and fix some bugs that we found during the testing on Linux, MacOS and Windows platforms.</p>
-<p>To run the new <a href="2024.12.01.html">BUDA applications</a>, our users need to have <a href="2024.12.01.html">Docker or Podman</a> installed on their systems. While this is quite easy to do on Linux and MacOS, it requires additional steps on Windows to install WSL (Windows Subsystem for Linux). As we don't want to leave any users behind, we are preparing a detailed guide to assist with the installation process including the custom pre-configured image that has basically one-step installation that could be downloaded from <a href="https://github.com/BOINC/boinc-buda-runner-wsl/releases">GitHub</a>. The complete guide is already awailable in the <a href="https://github.com/BOINC/boinc/wiki/Installing-Dockerhttps://github.com/BOINC/boinc/wiki/Installing-Docker">BOINC Wiki</a>.
+<p>To run the new <a href="2024.12.01.html">BUDA applications</a>, our users need to have <a href="2024.12.01.html">Docker or Podman</a> installed on their systems. While this is quite easy to do on Linux and MacOS, it requires additional steps on Windows to install WSL (Windows Subsystem for Linux). As we don't want to leave any users behind, we are preparing a detailed guide to assist with the installation process including the custom pre-configured image that has basically one-step installation that could be downloaded from <a href="https://github.com/BOINC/boinc-buda-runner-wsl/releases">GitHub</a>. The complete guide is already awailable in the <a href="https://github.com/BOINC/boinc/wiki/Installing-Docker">BOINC Wiki</a>.
```

Comment:
The link `<a href="2024.12.01.html">Docker or Podman</a>` also looks incorrect but I do not know what it should point to.